scripts/cat/thoughts.py

```diff
@@ -283,14 +278,15 @@ def load_thoughts(main_cat, other_cat, game_mode, biome, season, camp):
         else:
             life_dir = "dead"
 
-        if not main_cat.dead and main_cat.outside:
+        if main_cat.dead:
+            if main_cat.outside:
+                spec_dir = "/unknownresidence"
+            elif main_cat.df:
+                spec_dir = "/darkforest"
+            else:
+                spec_dir = "/starclan"
+        elif main_cat.outside:
```

Comment:
If we touch this code, I'd prefer if we used a path object instead, like `os.path`.

Reply:
I agree with you on that. But I did not feel it's my place to decide or make such design changes.

This is my first time contributing to open source and I don't want to overstep by doing more drastic changes

Reply:
All good, I don't blame you.

Reply:
After contemplating implementation of os.path (or equivalent alternative), I have concluded that it would be more logical to do so in a PR dedicated to it. Doing so in only 1 location (or a few) only muddle the consistency of the code.

I will therefore mark this as resolved, as I still think the code's intent is easier to read.